splitgpg2/__init__.py

```diff
@@ -284,7 +314,7 @@ def _parse_bool_val(self, value, option_name) -> bool:
         )
         raise ValueError(value)
 
-    def load_config(self, config) -> None:
+    def load_config(self, config: configparser.SectionProxy) -> None:
```

Comment:
pylint complains this function got too long, and indeed it isn't easy to follow. Maybe split subkeys-only keyring creation into separate function (which logically indeed is not really loading config)?